gitops/environments/envs/dev/external-secrets-wl.yaml

```diff
@@ -0,0 +1,112 @@
+---
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: ci-secrets
+  namespace: wl-<WL_NAME>-dev
+  annotations:
+    argocd.argoproj.io/sync-wave: '0'
+spec:
+  target:
+    name: ci-secrets
+  secretStoreRef:
+    kind: ClusterSecretStore
+    name: vault-kv-secret
+  refreshInterval: 10s
+  dataFrom:
+    - extract:
+        key: /ci-secrets
+---
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: argo-secrets
+  namespace: wl-<WL_NAME>-dev
+  annotations:
+    argocd.argoproj.io/sync-wave: '0'
+spec:
+  target:
+    name: argo-secrets
+  secretStoreRef:
+    kind: ClusterSecretStore
+    name: vault-kv-secret
+  refreshInterval: 10s
+  data:
+    - remoteRef:
+        key: oidc/argo
+        property: client_id
+      secretKey: client-id
+    - remoteRef:
+        key: oidc/argo
+        property: client_secret
+      secretKey: client-secret
```

Comment:
Is it really necessary to have OIDC secrets inside the workload? Why?

Reply:
99,9% you're right, I'll check it out